### ISSUE TITLE
Everything after css`...` helper failed to highlight

### ISSUE THREAD

**Issue by ImSolitude** [styled-components/vscode-styled-components#208]
I cannot use `css` helper function in any way, it always fail to highlight my css...

![after](https://user-images.githubusercontent.com/20565199/73693932-c7aee280-46df-11ea-83b9-030192b6fa7a.png)

am I missing something?

**Comment by mattfwood** [comment]
Been having the same issue for a while

**Comment by kolnogorov** [comment]
The same for me (looks like it's a common bug):
![image](https://user-images.githubusercontent.com/8288103/76325759-bf5a5080-62f8-11ea-87ee-9ff391cd3c7d.png)

But there's a workaround, you should add space between `css` and opening template literal:
![image](https://user-images.githubusercontent.com/8288103/76325871-e3b62d00-62f8-11ea-97e8-d062782aabc1.png)

(but autoformat breaks it back)

It's definitely a bug and should be fixed, i guess.

**Comment by junkboy0315** [comment]
@kolnogorov 
same here. thanks for workaround.

**Comment by rsilvamanayalle** [comment]
I have a "prettier" solution.. 

![ahuu](https://user-images.githubusercontent.com/12520426/79917872-f7bd7600-83f0-11ea-99c1-75235b896e3b.png)


![hohoh](https://user-images.githubusercontent.com/12520426/79917727-a614eb80-83f0-11ea-90ce-75134d5415bd.png)

**Comment by rsilvamanayalle** [comment]
I have a partial solution.

![hohoh](https://user-images.githubusercontent.com/12520426/79919007-289eaa80-83f3-11ea-8287-c125b850121c.png)

![ahuu](https://user-images.githubusercontent.com/12520426/79919041-32c0a900-83f3-11ea-9878-d954bbf20f56.png)

**Comment by junkboy0315** [comment]
Problems disappeard on my environment.
I don't know why, but here's what I did:

- Cmd+Shift+P ->`Disable all extensions` command
- uninstall vscode-styled-component
- Cmd+Shift+P -> `Enable all extensions` command
- re-install vscode-styled-component

**Comment by bexoss** [comment]
+1

**Comment by karlhorky** [comment]
Had this issue with highlighting Emotion code using `css` tagged template strings. Seems to have started after upgrading to VS Code May 2020 (version 1.46).

I disabled the `vscode-styled-jsx` extension as in the comment below, and the issue seems to have been resolved.

https://github.com/emotion-js/emotion/issues/1316#issuecomment-486122598

Unfortunately it looks like [`vscode-styled-jsx`](https://github.com/iFwu/vscode-styled-jsx) is unmaintained:

> This repository is no longer actively maintained
